Fix a compile error observed on jammy:

  FAILED: obj/content/test/test_support/prefetch_test_util.o
  [...]
  In file included from ../../content/public/test/prefetch_test_util.cc:5:
  ../../content/public/test/prefetch_test_util.h:19:5: error: constexpr variable 'InvalidPrefetchContainerIdForTesting' must be initialized by a constant expression
     19 |     InvalidPrefetchContainerIdForTesting = PrefetchContainerIdForTesting("");
        |     ^                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  /usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/stl_construct.h:97:14: note: construction of subobject of member '_M_local_buf' of union with no active member is not allowed in a constant expression
     97 |     { return ::new((void*)__location) _Tp(std::forward<_Args>(__args)...); }
        |              ^
  /usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/char_traits.h:264:6: note: in call to 'construct_at<char, const char &>(&InvalidPrefetchContainerIdForTesting.value_.union (anonymous union at /usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/basic_string.h:215:7)._M_local_buf[0], "".union (anonymous union at /usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/basic_string.h:215:7)._M_local_buf[0])'
    264 |             std::construct_at(__s1 + __i, __s2[__i]);
        |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  /usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/char_traits.h:433:11: note: in call to 'copy(&InvalidPrefetchContainerIdForTesting.value_.union (anonymous union at /usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/basic_string.h:215:7)._M_local_buf[0], &"".union (anonymous union at /usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/basic_string.h:215:7)._M_local_buf[0], 1)'
    433 |           return __gnu_cxx::char_traits<char_type>::copy(__s1, __s2, __n);
        |                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  /usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/basic_string.h:683:6: note: in call to 'copy(&InvalidPrefetchContainerIdForTesting.value_.union (anonymous union at /usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/basic_string.h:215:7)._M_local_buf[0], &"".union (anonymous union at /usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/basic_string.h:215:7)._M_local_buf[0], 1)'
    683 |             traits_type::copy(_M_local_buf, __str._M_local_buf,
        |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    684 |                               __str.length() + 1);
        |                               ~~~~~~~~~~~~~~~~~~~
  ../../base/types/strong_alias.h:85:9: note: in call to 'basic_string("")'
     85 |       : value_(std::move(v)) {}
        |         ^~~~~~~~~~~~~~~~~~~~
  ../../content/public/test/prefetch_test_util.h:19:44: note: in call to 'StrongAlias("")'
     19 |     InvalidPrefetchContainerIdForTesting = PrefetchContainerIdForTesting("");
        |                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

--- a/content/public/test/prefetch_test_util.h
+++ b/content/public/test/prefetch_test_util.h
@@ -15,7 +15,7 @@ namespace content::test {
 
 using PrefetchContainerIdForTesting =
     base::StrongAlias<class PrefetchContainerIdForTestingTag, std::string>;
-inline constexpr PrefetchContainerIdForTesting
+inline const PrefetchContainerIdForTesting
     InvalidPrefetchContainerIdForTesting = PrefetchContainerIdForTesting("");
 
 class TestPrefetchWatcherImpl;
